### PR TITLE
feat: add feature to allow registration of multiple clients #7725

### LINKED ISSUE #7725: feat(jans-tarp): add feature to allow registration of multiple clients
image

### PR DESCRIPTION
**Pull request by duttarnab**
closes #7725

### CODE REVIEW

File: demos/jans-tarp/src/options/authFlowInputs.tsx [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+  const [acrValueOption, setAcrValueOption] = React.useState("");
+  const [additionalParams, setAdditionalParams] = React.useState("");
+  const [loading, setLoading] = React.useState(false);
+  
+
+  React.useEffect(() => {
+    if (isOpen) {
+      handleOpen();
+    } else {
+      handleClose();
```
Comment (jv18creator) [inline demos/jans-tarp/src/options/authFlowInputs.tsx:44]: try to reset all the values/errors on close if possible. As of now when I close the dialog and open it again, I can see the issuer error.

File: demos/jans-tarp/webpack.common.js [hunk 2]
```diff
@@ -64,6 +64,22 @@ const chromeConfig = merge(commonConfig, {
                 from: path.resolve('src/static/icon.png'),
                 to: path.resolve('dist/chrome')
             },
+            {
```
Comment (jv18creator) [inline demos/jans-tarp/webpack.common.js]: If there are common images with the same path, could you please create a shared array of objects and try to append that array in both places?

**Review by jv18creator — CHANGES_REQUESTED** [review]
Please have a look at the inline comments.

File: demos/jans-tarp/src/options/authFlowInputs.tsx (hunk 1 above)
Comment (duttarnab) [reply demos/jans-tarp/src/options/authFlowInputs.tsx:44]: done

File: demos/jans-tarp/webpack.common.js (hunk 2 above)
Comment (duttarnab) [reply demos/jans-tarp/webpack.common.js]: done

**Review by moabu — APPROVED** [review]

**Pull request merged**